src/lib/orchestrator.ts

```diff
@@ -172,7 +172,7 @@ export async function listAssessmentResults(): Promise<AssessmentResult[]> {
  * @returns an array of {@link AssessmentResult}s.
  */
 export async function listCloudServiceAssessmentResults(serviceId: string, fetch = window.fetch): Promise<AssessmentResult[]> {
-    const apiUrl = clouditorize(`/v1/orchestrator/assessment_results?pageSize=1000&filteredCloudServiceId=${serviceId}`);
+    const apiUrl = clouditorize(`/v1/orchestrator/assessment_results?pageSize=1000&filteredCloudServiceId=${serviceId}&orderBy=name&asc=true`);
```

Comment:
Why is this one sorted by name?

Reply:
But these are results, not cloud services? Do they even have a name?

Reply:
You're absolutely right. I only read CloudService.

Reply:
I added the sorting also to the CloudServices, but that didn't work, so I deleted it again.